www/breakdownTimeline.php

```diff
@@ -3,305 +3,28 @@
-                $groupColors = array('Scripting' => '#f1c453',
-                                   'Layout' => '#9a7ee6',
-                                   'Painting' => '#71b363',
-                                   'Loading' => '#70a2e3',
-                                   'Other' => '#f16161',
-                                   'Idle' => '#cbd1d9');
-                if (!array_key_exists('Idle', $processing)) {
-                    $processing['Idle'] = 0;
-                }
-                foreach ($processing as $type => $time) {
-                    $group = 'Other';
-                    if (array_key_exists($type, $mapping)) {
-                        $group = $mapping[$type];
-                    }
-                    $groups[$group] += $time;
-                }
-            }
-            ?>
-
-<div class="results_main_contain">
-            <div class="results_main">
-
-            <div class="results_and_command">
-
-            <div class="results_header">
-                <h2>Main-thread Processing</h2>
-                <p>Where the browser's main thread was busy, not including idle time waiting for resources <?php
-                      echo " (<a href=\"/timeline/" . VER_TIMELINE . "timeline.php?test=$id&run=$run&cached=$cached\" title=\"View Chrome Dev Tools Timeline\">view timeline</a>)";
-                ?>.</p>
-            </div>
-
-
-            </div>
-
-            <div id="result" class="results_body">
-              <h3 class="hed_sub">Processing Breakdown</h3>
-
-
-
-          <div class="breakdownFrame">
-            <div class="breakdownFrame_item">
-              <div id="pieGroups" ></div>
-              <div class="table visualization_table" id="tableGroups" ></div>
-            </div>
-            <div class="breakdownFrame_item">
-              <div id="pieEvents" ></div>
-              <div class="table visualization_table" id="tableEvents"></div>
-            </div>
-          </div>
-
-          <h3 class="hed_sub">Timing Breakdown</h3>
-
-          <p>All of the main-thread activity including idle (waiting for resources usually) <?php
-                      echo " (<a href=\"/timeline/" . VER_TIMELINE . "timeline.php?test=$id&run=$run&cached=$cached\" title=\"View Chrome Dev Tools Timeline\">view timeline</a>)";
-            ?>.</p>
-
-
-          <div class="breakdownFrame">
-          <div class="breakdownFrame_item">
-            <div id="pieGroupsIdle" ></div>
-            <div class="table visualization_table" id="tableGroupsIdle"></div>
-          </div>
-          <div class="breakdownFrame_item">
-            <div id="pieEventsIdle"></div>
-            <div class="table visualization_table" id="tableEventsIdle"></div>
-          </div>
-          </div>
-          </div>
-
-
 
-        </div>
-        <?php include('footer.inc'); ?>
-        </div></div></div>
+define('NOBANNER', true); // Unclear if its needed
```

Comment:
I think it is. Do you have anything in your 'alert' setting?

https://github.com/WPO-Foundation/webpagetest/blob/d009915d76fa71d45384f5a8850ce9815fddbd68/www/header.inc#L50

if so, this is the banner that shows up on top, e.g.
<img width="1254" alt="Screen Shot 2022-10-21 at 12 05 39 PM" src="https://user-images.githubusercontent.com/51308/197240263-008cac93-a8e4-4d8c-8b5f-54995def879f.png">

Reply:
I didn't have anything like that in my banner but that is probably because i'm using my docker image.